app/ui-react/syndesis/src/modules/data/shared/ViewSqlForm.tsx

```diff
@@ -0,0 +1,192 @@
+import { WithVirtualizationHelpers } from '@syndesis/api';
+import { AutoForm, IFormDefinition } from '@syndesis/auto-form';
+import { QueryResults } from '@syndesis/models';
+import { Container } from '@syndesis/ui';
+// tslint:disable-next-line:no-implicit-dependencies
```

Comment:
dont think you need this tsLint line?...

Reply:
see below